client/config/crd/snapshot.storage.k8s.io_volumesnapshotclasses.yaml

```diff
@@ -102,7 +116,8 @@ spec:
           parameters:
             additionalProperties:
               type: string
-            description: parameters is a key-value map with storage driver specific parameters for creating snapshots. These values are opaque to Kubernetes.
+            description: parameters is a key-value map with storage driver specific
+              parameters for creating snapshots. These values are opaque to Kubernetes.
```

Comment:
This indentation change should be applied to line 71.  Please leave v1beta1 section unchanged, including indentation changes.

Reply:
Done